fe1-web/src/features/evoting/objects/__tests__/utils.ts

```diff
@@ -0,0 +1,157 @@
+import { EventTags, Hash, Timestamp } from 'core/objects';
+import { CastVote, ElectionResult, EndElection } from 'features/evoting/network/messages';
+import STRINGS from 'resources/strings';
+import { mockLaoId, mockLaoIdHash } from '__tests__/utils';
+import { Election, ElectionStatus, Question, RegisteredVote, Vote } from '../Election';
+
+const TIMESTAMP = new Timestamp(1609455600); // 1st january 2021
+const VERSION = STRINGS.election_version_identifier;
+const CLOSE_TIMESTAMP = new Timestamp(1609542000); // 2nd january 2021
+
+export const mockElectionName = 'An election';
```

Comment:
This file is a great addition, thanks ! But I'll probably put it higher in the folders (closer to `evoting`). The problem being that all tests of the feature will use these constants, and here they're only in `objects`. I can see the logic behind it, but it might be better to put it somewhere else, I'll let you figure it out.